tracer/test/snapshots/StackExchangeRedisTests.V1_2_0.SchemaV1.verified.txt

```diff
@@ -2783,13 +2985,15 @@ at StackExchange.Redis.ConnectionMultiplexer.ExecuteSyncImpl[T](Message message,
     Tags: {
       component: StackExchangeRedis,
       env: integration_tests,
-      out.host: stackexchangeredis,
-      out.port: 6390,
+      out.host: stackexchangeredis-replica,
```

Comment:
Didn't expect that one to change, but maybe that's just that some times we can hit the repica? 🤷 Have you checked? That said looking at your code changes I'm not worried at all

Reply:
Just tried generating it on master and it makes this exact change as well, it probably hasn't been run for a long time 🤔

Reply:
+1, I think this snapshot has just been out of date since it is not run regularly. Other snapshot files also have the line `out.host: stackexchangeredis-replica,`